src/styles/index.scss

```diff
@@ -1,3 +1,71 @@
+html {
+  height: 100%;
+}
+
 body {
   margin: 0;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  position: relative;
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+
+  &__minutes {
+    position: absolute;
+    left: calc(50% - 1.5vmin);
+    top: calc(50% - 20vmin);
+    content: '';
+    display: block;
+    width: 3vmin;
+    height: 20vmin;
+    background-color: #0700ff;
+    animation: nev 3600s infinite steps(60);
+    transform-origin: bottom center;
+  }
+
+  &__minutes--speed-up {
+    animation-duration: 600s;
+  }
+
+  &__sekundes {
```

Comment:
There seems to be a spelling mistake in your class name. Did you mean `__seconds` instead of `__sekundes`?